documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -31,19 +31,23 @@ kind: Secret
 metadata:
   name: <secret>
   namespace: {namespace}
+  labels:
+    createdForProviderType: <provider_type> <1>
 type: Opaque
 stringData:
-  user: <user> <1>
-  password: <password> <2>
-  cacert: | <3>
-    <engine_ca_certificate>
-  thumbprint: <vcenter_fingerprint> <4>
+  user: <user> <2>
+  password: <password> <3>
+  cacert: | <4>
+    <ca_certificate>
+  url: <api_end_point>
+  thumbprint: <vcenter_fingerprint> <5>
 EOF
 ----
-<1> Specify the vCenter user or the {rhv-short} {manager} user.
-<2> Specify the user password.
-<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
-<4> VMware only: Specify the vCenter SHA-1 fingerprint.
+<1> Supports verifying the secret when a third-party certificate is used. Allowed values are `ovirt`, `vsphere`, and `openstack`.
```

Comment:
that's a bit inaccurate, it's not about verifying the secret when a third-party certificate is used but rather to verify the credentials are correct when the remote system is accessible *and* retrieve the Manager CA when third-party certificate is specified